apps/tx-processor/src/deposits/index.test.ts

```diff
@@ -1,15 +1,17 @@
 import { processUnsettledDeposits } from '.'
 import { init } from '@thecointech/firestore/mock';
 import { GetContract } from '@thecointech/contract';
-import { getCurrentState } from '../statemachine/types';
+import { getCurrentState } from '@thecointech/tx-processing';
+import { RbcApi } from '@thecointech/rbcapi';
 
 it("Can complete deposits", async () => {
 
   jest.setTimeout(900000);
   init({});
 
   const theContract = await GetContract();
-  const deposits = await processUnsettledDeposits(theContract);
+  const bank = new RbcApi();
```

Comment:
This is mocked, but I would actually kinda prefer it was clearer.

It makes me feel nervous to see my testing code running with a potential connection to my bank account